packages/flutter/lib/src/cupertino/tab.dart

```diff
@@ -0,0 +1,161 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'route.dart';
+
+/// A single tab with its own [Navigator] state and history.
+///
+/// A typical tab used as the content of each tab in a [CupertinoTabScaffold]
```

Comment:
this seems to confuse two meaning of "tab". @HansMuller might have suggestions for names to contrast the actual tab control that brings up the page, and the view that is associated with that tab.

Reply:
Ya, the names do collide... 

Well I think the material widgets have precedence. Wholesale renamed CupertinoTab to CupertinoTabView to match. 

So tab bar is tab bar
tab bar item is just 'tab'
and the tab pages are tab views.